js/src/block-editor/controls/multiselect.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
```

Comment:
Adding this boilerplate to every file isn't ideal.

But it lets VS Code to type-checking, which has revealed some issues, like with the `multiple` prop below.